pkg/lib/cockpit-components-dialog.jsx

```diff
@@ -302,20 +308,24 @@
     const dialogObj = { };
     let error = null;
     dialogObj.props = props;
+    dialogObj.props.set_error = e => {
+        error = typeof e === 'object' && e !== null ? (e.message || e.toString()) : e;
+        dialogObj.render();
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19200-20230821-124816-2fdc0a03-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)